toqito/channel_metrics/diamond_norm.py

```diff
@@ -38,8 +39,8 @@ def diamond_norm(choi_1: np.ndarray, choi_2: np.ndarray) -> float:
     >>> choi_dephasing = dephasing(dim=2)
     >>> choi_identity = np.identity(2**2)
     >>> dn = diamond_norm(choi_dephasing, choi_identity)
-    >>> print("Diamond norm between dephasing and identity channels: ", dn)
-    Diamond norm between depolarizing and identity channels:  -3.5989938811978797e-09
+    >>> print("Diamond norm between dephasing and identity channels: ", '%.2f' % dn)
+    Diamond norm between dephasing and identity channels:  -0.00
```

Comment:
Do we want to keep the precision on L42 and L30? Or the new changes are fine as a demo?